Cargo.toml

```
@@ -20,7 +20,7 @@ citadel_workspace_lib = { path = "./citadel_workspace_lib", default-features = f
 
 # standard deps
 serde = { version = "1.0.104", features = ["derive"] }
-citadel_sdk = { git = "https://github.com/Avarok-Cybersecurity/Citadel-Protocol", branch = "master"}
+citadel_sdk = { git = "https://github.com/Avarok-Cybersecurity/Citadel-Protocol", branch = "group_channel_update" }
```

The PR looks good! Once we merge in your PR in the protocol, we can go ahead and make this adjustment and finish Group functionality.

```suggestion
citadel_sdk = { git = "https://github.com/Avarok-Cybersecurity/Citadel-Protocol" }
```